.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "none"
+}
```

Comment:
I don't think this file should be added.

Reply:
Will remove it.